tigerpath/templates/tigerpath/about.html

```diff
@@ -35,118 +35,34 @@ <h1 class="mb-3">About TigerPath</h1>
-          </ul>
-
-          <p class="tl-section">April 22nd:</p>
-          <ul>
-            <li><p>Clean up UI and make it look nicer, if it doesn’t already look good</p></li>
-            <li><p>Let the user customize requirements that they have fulfilled through other means (such as study abroad)</p></li>
-            <li><p>Inform the user of when courses are typically offered (such as how frequently the course is offered and which semester it is typically offered in)</p></li>
-            <li><p>Start putting in requirements for other majors</p></li>
-          </ul>
-
-          <p class="tl-section">April 27th: Alpha test</p>
-          <ul>
-            <li><p>By this time, we should have full basic functionality of the app, including the following:</p></li>
-            <li><p>Being able to retrieve saved user data during a later session</p></li>
-            <li><p>Having a presentable UI</p></li>
-            <li><p>Requirements customization and course time predictions</p></li>
-            <li><p>Continue propagating app for people to try out</p></li>
-          </ul>
-
-          <p class="tl-section">April 29th:</p>
-          <ul>
-            <li><p>Continue putting in requirements for other majors</p></li>
-            <li><p>Work on other additional features</p></li>
-            <li><p>Start working on presentation/final documentation</p></li>
-          </ul>
-
-          <p class="tl-section">May 3rd: Beta Test</p>
-          <ul>
-            <li><p>The app is fully usable and useful to a user that would want to use it</p></li>
-            <li><p>Do a beta test release to the Princeton community</p></li>
-          </ul>
+    <section class="container mb-4">
+      <div class="text-section">
+        <h4>Description</h4>
+        <p>TigerPath is a web application that helps Princeton University students plan out their four-year course schedules.</p>
+        <p>There is currently no good way for students to manage what courses they’re taking and when they're taking them beyond the current or upcoming semester. Many students try to manually do this by using an Excel spreadsheet or pen-and-paper, but any mistake can easily lead to an overloaded semester or a need to take summer classes in order to fulfill missed requirements.</p>
+        <p>TigerPath aims to solve this problem by providing a platform for students to manage what courses they're taking during each semester and giving them feedback on the major requirements their schedule fulfills and does not fulfill. Our goal is for TigerPath to become a useful, enduring tool that ensures Princeton students are able to stay on top of requirements and graduate on time.</p>
+        <p>We currently fully support COS (AB and BSE), ELE, CBE, and Woody Woo, but we are continually adding more majors. If you'd like to be notified when your major is added to TigerPath, please fill out <a href="https://goo.gl/forms/pKxjmubIOSCOeR8L2">this form</a>!</p>
```

Comment:
Why is it that they have to login to access the form?

Reply:
We need to capture their email address so we can send them an email

Reply:
It might be turning away people who would like to request their major be added, but are unsure about providing their email addresses.
I think providing an email for us to contact them should be optional, so that we can also gauge interest with people who don't want to be contacted.

Reply:
As in. people can no longer request new major support through the feedback form (or at least they're told not to do so), and then when they come upon this form, they're asked to log in, which seems too much, so they might just give up and we won't know.

Reply:
You can change it in the feedback form, it's not relevant to this pull request